### PR TITLE
Data track util

### PR DESCRIPTION
**Pull request by ozerovs**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   "homepage": "https://nebenan.de/",
   "repository": "good-hood-gmbh/nebenan-components",
   "bugs": "https://github.com/good-hood-gmbh/nebenan-components/issues",
-  "version": "9.7.2",
+  "version": "9.7.3",
```
Comment (peterhass) [inline package.json:10]: it's a new feature -> 9.8

File: src/context_list/utils.jsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const DATA_TRACK_REGEX = /^data-track[^\s]*/;
+
+export const parseDataTrack = (options) => Object.keys(options).reduce((acc, elem) => {
```
Comment (peterhass) [inline src/context_list/utils.jsx:3]: `getTrackingAttributes` ?

File: test/context_list/utils.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+    const options = {
+      ...result,
+      bird: 'animal',
+      data: 'value',
+      'some-random-data-track': 'value',
+      'data-tracright': 'kek',
+    };
+
+    assert.deepEqual(parseDataTrack(options), result);
```
Comment (peterhass) [inline test/context_list/utils.js:30]: I would inline `options` and `result` - I try to have as little logic in tests as possible

File: src/context_list/index.jsx [hunk 4]
```diff
@@ -149,9 +151,8 @@ class ContextList extends PureComponent {
     const onClick = this.handleClick.bind(this, key);
     const onMouseEnter = this.handleMouseEnter.bind(this, key);
     const children = getOption(key, this.props.options);
-    const dataTrack = this.props.options[key]['data-track'];
-    const cleanProps = { key, className, onClick, onMouseEnter };
-    if (dataTrack) cleanProps['data-track'] = dataTrack;
+    const dataTrack = parseDataTrack(this.props.options[key]);
```
Comment (peterhass) [inline src/context_list/index.jsx:154]: `const trackingAttributes = getTrackingAttributes(this.props.options[key]);` ?

**Review by peterhass — APPROVED** [review]

File: test/context_list/utils.js (hunk 3 above)
Comment (ozerovs) [reply test/context_list/utils.js:30]: Isn't this way more readable?

Comment (peterhass) [reply test/context_list/utils.js:30]: It's not too bad if you span it over multiple lines. I like it better when you instantly see what is happening 

- assert.deepEqual 
- - function call
- - - function args
- - expected

Not sure if the others see it the same way but even if you don't want to inline: Don't merge result into options. Hardcode it.

**Review by peterhass — APPROVED** [review]

**Pull request merged**